tests/lib/rules/indent.js

```diff
@@ -5724,6 +5724,105 @@ ruleTester.run("indent", rule, {
             `,
             options: [4, { MemberExpression: 2 }],
             parserOptions: { ecmaVersion: 2015 }
+        },
+        {
+            code: unIndent`
+                const foo = async (arg1,
+                                   arg2) =>
+                {
+                  return arg1 + arg2;
+                }
+    `,
```

Comment:
It would be nice to consistently align closing backticks with `code`/`output`, like in all previously existing tests.